src/Rule/CompareTo.php

```diff
@@ -129,10 +129,6 @@ protected function validateValue($value, ValidationContext $context = null): Res
     {
         $result = new Result();
 
-        if ($this->compareValue === null) {
```

Comment:
Why?

Reply:
Because `null` is an allowable value too.